common/buildings/wc_cultural_tradepost.txt

```diff
@@ -0,0 +1,314 @@
+trade_post = {
+	
+	############CULTURAL#################
+	
+	### General Building
+	tp_road_tavern = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { not_tradepost_culture_groups = yes }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 275
+		build_time = 180
+		ai_creation_factor = 95
+		
+		garrison_size = 0.5
+		tradevalue = 20
+		trade_route_wealth = 1

```

Comment:
[The Spreadsheet](https://docs.google.com/spreadsheets/d/1KLHQRAANjXuKB1PYqk-aJu_60DV_KCLXiyZjJV7mwNU/edit?usp=sharing) says it should be...
```suggestion
		trade_route_wealth = 0.2
```